neptune/new/internal/backends/utils.py

```diff
@@ -183,24 +183,24 @@ def verify_client_version(client_config: ClientConfig, version: Version):
         client_config.version_info.min_compatible
         and client_config.version_info.min_compatible > version
     ):
-        raise UnsupportedClientVersion(
+        raise NeptuneClientUpgradeRequiredError(
             version, min_version=client_config.version_info.min_compatible
         )
     if (
         client_config.version_info.max_compatible
         and client_config.version_info.max_compatible < version_with_patch_0
     ):
-        raise UnsupportedClientVersion(
+        raise NeptuneClientUpgradeRequiredError(
             version, max_version=client_config.version_info.max_compatible
         )
     if (
         client_config.version_info.min_recommended
         and client_config.version_info.min_recommended > version
     ):
         click.echo(
-            "WARNING: We recommend an upgrade to a new version of neptune-client - {} (installed - {}).".format(
-                client_config.version_info.min_recommended, version
-            ),
+            f"WARNING: Your version of Neptune client library ({version}) is deprecated,"
```

Comment:
Missing header
```
{h1}
----NeptuneClientUpgradeRecommended-------------------------------------------------------------
{end}
````

Reply:
Requesting @Herudaio to comment further on this. Personally, I'm not yet sure where we should or should not have H1s.

Reply:
There seems to be no strong preference either way, so I think we should leave out the H1 for now.